### ISSUE TITLE
recent soundcast run had a strange average transit trip length and bike trip destinations

### ISSUE THREAD

**Issue by Ennazus** [psrc/soundcast#42]
_From @Ennazus on July 31, 2014 22:10_

see the run on ModelSrv1\soundcat_new

_Copied from original issue: psrc/soundcast#146_

**Comment by Ennazus** [comment]
Here’s some things Brice and I figured out today about the transit and bike distances.

The transit trip lengths are incorrect in the model summary.  The values may be weighted.  We are unsure why they are inaccurate. But the values in the fields travdist and travtime are much longer than occur in google maps.  I have another issue where we will work on directly tagging the times from the skims, to fix this.

We determined this by sampling several locations and checking them in google maps, the field travdist from Daysim, and the skims.

As a temporary fix, I just noted the problem in the data summaries.
So the transit problem is not actually a problem in the model but in the data field that is a post-process step.

However, the bike distances are definitely still too long.
What appears to be happening is that people are very very attracted to biking downtown and in Bellevue.  So they go too far to bike there.

To fix this, we should: 
1. possibly increase the weight on bike time in the Daysim configuration,.
2. decrease the coefficients on mixed use density, employment density, and intersection density in the mode choice models.
3. look into changing the cost coefficient in the mode choice models.  It may be that the models are too sensitive to parking cost. We saw when the scale of parking costs were off.
The transit mode choice model was very very sensitive to parking costs.  When the parking costs were zero, transit trips were halved.
